Applications/ProductionManager/Contracts/Predict/CuttingPredictionRequest.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Runtime.Serialization;
+
+namespace HomagConnect.ProductionManager.Contracts.Predict;
+
+/// <summary>
+/// Cutting Prediction Request
+/// </summary>
+public class CuttingPredictionRequest : IExtensibleDataObject
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="CuttingPredictionRequest"/> class.
+    /// </summary>
+    public CuttingPredictionRequest()
+    {
+        
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="CuttingPredictionRequest"/> class.
+    /// </summary>
+    public CuttingPredictionRequest(IEnumerable<CuttingPredictionPart>? productionEntities):this()
+    {
+        ProductionEntities = productionEntities;
+    }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="CuttingPredictionRequest"/> class.
+    /// </summary>
+    public CuttingPredictionRequest(IEnumerable<CuttingPredictionPart>? productionEntities, string? machine) :this(productionEntities)
```

Comment:
"machine" vs. "machineNumber". The HOMAG-Machine-number is not known by most customers. Maybe we need to add a method which provides all known machines (number + tapio displayname).

Reply:
something like "subscriptionInfo". To make it really useful, we should also add the licenses to each machine, so the customer knows, which machine is attached to hich app